java/client/src/org/openqa/selenium/chrome/ChromeDriverService.java

```diff
@@ -56,13 +56,6 @@
   public static final String CHROME_DRIVER_APPEND_LOG_PROPERTY =
       "webdriver.chrome.appendLog";
 
-  /**
-   * Boolean system property that defines whether the chromedriver executable should be started
-   * with verbose logging.
-   */
-  public static final String CHROME_DRIVER_VERBOSE_LOG_PROPERTY =
```

Comment:
This is part of the public API of the language binding, so it's not safe to remove this without deprecating it first. I think the best way to handle this is to check for a `CHOME_DRIVER_LOG_LEVEL_PROPERTY` and have that take precedence if set.